### ISSUE TITLE
iteratec Logo in main README.md is broken

### ISSUE THREAD

**Issue by o1oo11oo** [secureCodeBox/secureCodeBox#2026]
### Describe the bug

The iteratec logo that should be shown under sponsors in the main `README.md` file only shows the alt text.

### Steps To Reproduce

Go to https://github.com/secureCodeBox/secureCodeBox/tree/67ceaa9ff163052e18b154c613561435b0b5043e#sponsors (current most recent commit)

### Additional context

The image was deleted in commit f2eefbc93e45b4c7d3e268f71b58098682be757b, which is part of #1950 and relates to #942.
